src/ast/ast_validations.c

```diff
@@ -1681,6 +1681,10 @@ static AST_Validation _AST_Validate_ParseResultRoot(const cypher_parse_result_t
 		}
 	}
 
+	if(nroots == 0) {
```

Comment:
Please add a comment giving example(s) for queries which will set this condition to true